com.sigil_ebook.Sigil.json

```diff
@@ -0,0 +1,177 @@
+{
+    "app-id": "com.sigil_ebook.Sigil",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "command": "sigil",
+    "finish-args": [
+        "--socket=x11",
+        "--filesystem=home:rw"
+    ],
+    "modules": [
+        {
+            "name": "python3-pip",
```

Comment:
Any reason of install pip here?

Reply:
The [installation documentation](https://github.com/Sigil-Ebook/Sigil/blob/master/docs/Building_on_Linux.md#getting-python-34-or-higher) suggests installing `python3-pip`. I'm not sure if Sigil uses any functionality of pip as a library. I added this just to be sure, it can probably be removed.

Reply:
Any inputs from Sigil developers?

Reply:
On macOS our app id is exactly our web address: com.sigil-ebook.Sigil no undersvore.
pip3 is not required if all of the python3 packages needed for Sigil plugin support already exist.

Reply:
@kevinhendricks Unfortunately Flatpak doesn't allow dashes in the app-id because the DBus standard doesn't either. I agree it should be `com.sigil-ebook.Sigil`.